src/main/resources/application.yml

```diff
@@ -182,6 +182,7 @@ email:
   from: ${EMAIL_FROM}
 
 redisson:
+  shutdown-on-stop: false
```

Comment:
I remember you talking a little about this. What was this configuration change addressing again?

Reply:
The original method of injecting dependency into the entity and action instances was breaking the redis connection in nested dependent services, and this config change was part of an initial effort to create a manager to handle the redis connection, but that effort was abandoned since a better way of constructing the instances with factories was used, instead.

I removed the setting.